.taskcluster.yml

```diff
@@ -26,351 +26,343 @@ tasks:
       then: ${event.pull_request.head.repo.html_url}
       else: ${event.repository.html_url}
   in:
-    $if: 'tasks_for in ["github-pull-request", "github-push"]'
-    then:
-      - taskId: {$eval: as_slugid("lint_task")}
-        created: {$fromNow: ''}
-        deadline: {$fromNow: '1 hour'}
+    $let:
+      default_task_definition:
         provisionerId: aws-provisioner-v1
         workerType: github-worker
-        payload:
-          maxRunTime: 3600
-          image: python:3.7
-          command:
-            - "/bin/bash"
-            - "-lcx"
-            - "git clone --quiet ${repository} &&
-               cd bugbug &&
-               git -c advice.detachedHead=false checkout ${head_rev} &&
-               pip install --quiet -r test-requirements.txt &&
-               pre-commit run -a" # TODO Be smarter and run only on modified files
         metadata:
-          name: bugbug lint
-          description: bugbug lint
           owner: ${user}@users.noreply.github.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
+    in:
+      $if: 'tasks_for in ["github-pull-request", "github-push"]'
```

Comment:
probably the array should start from here.

An array in yaml is something like:
```
- element1
- element2
- element3
```

Or:
```
- taskId: val1
  created: val1
  ...
- taskId: val2
  created: val2
  ...
```

Reply:
@marco-c  you mean I should use `tasks` variable after Line-36 instead of Line-4 ?

Reply:
No, I mean you should do:
```
- $if: 'tasks_for in 
```
instead of
```
$if: 'tasks_for in 
```